### ISSUE TITLE
Node.js and copyleft licenses

### ISSUE THREAD

**Issue by thernstig** [nodejs/help#3892]
### Details

**TLDR:** Does projects using Node.js fall under copyleft licenses, such as GPL?

Looking at https://github.com/nodejs/node/blob/main/LICENSE it states that some parts of external libraries use *GNU General Public License (GPL)*. These are copyleft licenses.

As an example, check https://github.com/nodejs/node/blob/5fad0b93667ffc6e4def52996b9529ac99b26319/deps/uv/m4/libuv-check-flags.m4 (libuv is listed as a library Node.js depends on, see https://nodejs.org/en/docs/meta/topics/dependencies/).

So does this mean anyone using Node.js has to share their source code with anyone?

### Node.js version

Not applicable.

### Operating system

Not applicable.

### Scope

Not applicable.

### Module and version

Not applicable.

**Comment by F3n67u** [comment]
@thernstig Could you please post this question on [nodejs/node discussion](https://github.com/nodejs/node/discussions)? 

I think it will be more appropriate to post this question on nodejs/repo because it is more about node core code.

**Comment by thernstig** [comment]
Done! Thanks @F3n67u 

https://github.com/nodejs/node/discussions/43494
